data-model/src/main/java/io/micronaut/data/model/query/builder/jpa/JpaQueryBuilder.java

```diff
@@ -51,6 +54,28 @@
  */
 @Internal
 public class JpaQueryBuilder extends AbstractSqlLikeQueryBuilder implements QueryBuilder {
+
+    private static final NamingStrategy JPA_NAMING_STRATEGY = new NamingStrategy() {
```

Comment:
@radovanradic In the future it would be nice to you this kind of custom strategy instead of `computePropertyPaths` checks

Reply:
Yes, makes sense